presto-main/src/test/java/com/facebook/presto/operator/TestFileFragmentResultCacheManager.java

```diff
@@ -133,18 +157,40 @@ public void testBasic()
         assertEquals(stats.getCacheRemoval(), 2);
     }
 
-    private static void assertPagesEqual(Iterator<Page> pages1, Iterator<Page> pages2)
+    @Test
+    public void testThreadWrite()
     {
-        while (pages1.hasNext() && pages2.hasNext()) {
-            Page page1 = pages1.next();
-            Page page2 = pages2.next();
-            assertEquals(page1.getChannelCount(), page2.getChannelCount());
-            for (int i = 0; i < page1.getChannelCount(); i++) {
-                assertTrue(page1.getBlock(i).equals(0, 0, page2.getBlock(i), 0, 0, page1.getBlock(0).getSliceLength(0)));
-            }
+        ImmutableList.Builder<Future<String>> futures = ImmutableList.builder();
+        int i = 0;
+        int threadCnt = 10;
+        while (i < threadCnt) {
+            Future<String> future = writeExecutor.submit(() -> {
+                try {
+                    System.out.println(Thread.currentThread().getName());
+                    String threadInfo = String.format(WRITE_THREAD_NAME_FORMAT, Thread.currentThread().getName(), Thread.currentThread().getId());
```

Comment:
Curious: is `%s,%s` enough?

Reply:
i name it test write content,thread %s,%s,that i think it more clear